apps/web/src/pages/api/auth/[...nextauth].ts

```diff
@@ -1,43 +1,33 @@
 import NextAuth from 'next-auth';
 import TwitterProvider from 'next-auth/providers/twitter';
 import { config } from '../../../config';
-import { AccountProfile } from 'types/Score';
-import { userScore } from '../userScore';
 
 export default NextAuth({
   callbacks: {
-    async signIn({ account, profile }: AccountProfile) {
-      if (account.provider === 'twitter') {
-        const score = await userScore({ account, profile });
-        console.log(score);
-      }
-
-      return true;
-    },
     session({ session, token }) {
       session.id = token.sub;
+      session.access_token = token.access_token;
+      session.username = token.username;
+
       return session;
     },
+    async jwt({ token, account, profile }) {
+      if (profile != undefined) {
+        const { data }: any = profile;
+        const { username } = data;
+        token.username = username as string;
+      }
+      if (account) {
+        token.access_token = account.access_token;
+      }
+      return token;
+    },
   },
   providers: [
     TwitterProvider({
       clientId: config.TWITTER_CLIENT_ID,
       clientSecret: config.TWITTER_CLIENT_SECRET,
       version: '2.0',
```

Comment:
no more userinfo ?

Reply:
It has a default, you can see here: https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/src/providers/twitter.ts
![image](https://user-images.githubusercontent.com/5343791/159577804-0404a748-3889-4efd-b0e4-fa643a2535cf.png)

I think we can put userInfo again if we need to use it for something